content/guides/developer-guides/object-lifecycle-management/_index.md

```diff
@@ -0,0 +1,85 @@
+}
+```
+### Points to remember:
+    a) After unmarshalling into json struct and populating the fields, call the respective function from the s3 client and also in database.
+    b) Validate all the functions using log messages.
+    c) The API should successfully return and all the changes to be reflected in database should be seen in database client(for lifecycle feature the database is MongoDB)
+
+## How to start multi-cloud into the Docker container
+To start testing multi-cloud lifecycle feature, you need to first start multi-cloud service using steps below in the docker container.
+
+```
+docker-compose up -d
+docker ps
+```
+#### Check if multi-cloud api service and mongo is running:
+```
+CONTAINER ID        IMAGE                             COMMAND                  CREATED             STATUS              PORTS                                                NAMES
+c245b39b9a7d        opensdsio/multi-cloud-dataflow    "/dataflow"              6 hours ago         Up 6 hours                                                               multi-cloud_dataflow_1_b4432dc97238
+6bff89fd48b6        opensdsio/multi-cloud-datamover   "/datamover"             6 hours ago         Up 6 hours                                                               multi-cloud_datamover_1_6aa7bb101f33
+561d334d2ca8        opensdsio/multi-cloud-s3          "/s3"                    6 hours ago         Up 6 hours                                                               multi-cloud_s3_1_84b4c3f135b2
+a7288d668842        opensdsio/multi-cloud-api         "/api"                   6 hours ago         Up 6 hours          0.0.0.0:8089->8089/tcp                               multi-cloud_api_1_783889b02933
+247d70f62574        wurstmeister/kafka:2.11-2.0.1     "start-kafka.sh"         12 days ago         Up 6 hours          0.0.0.0:9092->9092/tcp                               multi-cloud_kafka_1_e666cc1a0f3b
+a2c136ae7421        opensdsio/multi-cloud-backend     "/backend"               12 days ago         Up 6 hours                                                               multi-cloud_backend_1_1983cccd8ac9
+eea40a48de97        wurstmeister/zookeeper            "/bin/sh -c '/usr/sb…"   12 days ago         Up 6 hours          22/tcp, 2888/tcp, 3888/tcp, 0.0.0.0:2181->2181/tcp   multi-cloud_zookeeper_1_6643a83ddcdd
+5dbac9cca10e        mongo                             "docker-entrypoint.s…"   12 days ago         Up 6 hours          0.0.0.0:27017->27017/tcp                             multi-cloud_datastore_1_a0281d684ef3
+```
+
+To check if the API changes are reflected in database, install mongoDB compass and check the metadata of the buckets
+
+![mongo connection image  ](opensds_mongo.png?raw=true)
+
+#### The postman request for API call:
+
+Here is a sample API call using postman, it contains request body information as well as response.
+
+![postman api call image  ](postman.png?raw=true)
```

Comment:
nit: The postman API call,  request and response samples can be added as a code block instead of a screenshot.